docs/content/en/docs/intro/_index.md

```diff
@@ -4,27 +4,25 @@ description: An introduction to Keptn and the usecases.
 weight: 10
 ---
 
-Keptn implements observability for deployments.
-It seamlessly integrates with deployment tools
+Keptn integrates seamlessly with cloud-native deployment tools
 such as ArgoCD, Flux, and Gitlab
 to bring application awareness to your Kubernetes cluster.
+Keptn supplements the standard deployment tools
+with facilities to ensure that your deployment is usable.
```

Comment:
This seems like a sentence that doesn't really covey anything meaningful. Shall we maybe change it to:
```suggestion
Keptn supplements the standard deployment tools
with features to help you ensure that your deployments are in
a healthy state.
```